ShowResult.py

```diff
@@ -0,0 +1,68 @@
+# Month names dictionary
+month = {
+    '1':'Jan',
+    '2':'Feb', 
+    '3':'Mar', 
+    '4':'Apr', 
+    '5':'May', 
+    '6':'Jun', 
+    '7':'Jul', 
+    '8':'Aug', 
+    '9':'Sep', 
+    '10':'Oct', 
+    '11':'Nov', 
+    '12':'Dec'
+}
+
+class ShowReult:
+    """Results class is used to save result and show them as per required format"""
+    def __init__(self,  highest = None, lowest = None, humidity = None):
+        self.highest = highest         
+        self.lowest = lowest
+        self.humidity = humidity
+
+    def chart_set(self,  chart):
+        """Set the chart list
+
+        Args:
+            chart ([list]): [list that contain tuple of month# max and min temprature resp.]
+        """
+        self.chart = chart
+    
+    def max_min_show(self):
+        """Show the highest, lowest temprature resp. and most humidity in required format"""
```

Comment:
Code reviewer don't know what is the required format. So add the format or example what is it doing